build.gradle

```diff
@@ -1,3 +1,12 @@
+buildscript {
+    repositories {
+        mavenCentral()
+    }
+    dependencies {
+        classpath 'biz.aQute.bnd:biz.aQute.bnd.gradle:4.2.0'
```

Comment:
I think the plugin can also be applied in the `plugins` block
`id 'biz.aQute.bnd.builder' version '4.2.0'`

Reply:
I'm not very familiar with gradle (to be honest, I've never worked with it), but it won't work for my local build if I demove that section. I applied it in the plugins-block.

Reply:
This is due to https://github.com/bndtools/bnd/issues/3174. After 4.3.0 is released the block can be removed again.